tests/test_language_extract_go.py

```diff
@@ -42,10 +42,10 @@ def test_data_string_offset(request, string, offset, encoding, go_strings):
     [
         # .text:0048B12F 8D 05 F8 08 4C 00 lea     eax, off_4C08F8 ; "hello world"
         # .text:0048B135 89 44 24 24       mov     [esp+28h+var_4], eax
-        pytest.param("hello world", 0x348f8, StringEncoding.UTF8, "go_strings32"),
+        pytest.param("hello world", 0x348F8, StringEncoding.UTF8, "go_strings32"),
```

Comment:
sorry, if that wasn't clear, I meant raw file offset, so when I check this address in the binary, there's the string

here it's at 0xA03E1 (so we need to add the section offset)

<img width="596" alt="2023-06-23_07-58-51_010Editor" src="https://github.com/mandiant/flare-floss/assets/17606537/bfec3b5d-92b3-4214-8691-e9a809b89be6">